### PR TITLE
[routing] change doc to best practices

### PR DESCRIPTION
**Pull request by dbu**

### CODE REVIEW

File: bundles/routing/configuration.rst [hunk 1]
```diff
@@ -125,7 +125,7 @@ generic_controller
 
 The controller to use when the route enhancers only determined the template but
 no explicit controller. The value is the name of a controller using either the
-``AcmeDemoBundle::Page::index`` or ``acme_demo.controller.page:indexAction``
+``AppBundle::Page::index`` or ``app.page_controller:indexAction``
 notation.
```
Comment (wouterj) [inline bundles/routing/configuration.rst]: Well, actually, all controller formats are supported right? So maybe we shouldn't explicitely mention these 2 (as you can also do `FQCN::methodName`)

File: bundles/routing/configuration.rst [hunk 2]
```diff
@@ -155,34 +155,36 @@ type:
 
     .. code-block:: yaml
 
+        # app/config/config.yml
         cmf_routing:
             dynamic:
                 controllers_by_type:
-                    editable: acme_main.controller:editableAction
+                    editable: app.cms_controller:editableAction
```
Comment (wouterj) [inline bundles/routing/configuration.rst]: As best practices don't suggest using controllers as services, what about using `AppBundle:Cms:editable` here?

File: bundles/routing/configuration.rst [hunk 3]
```diff
@@ -155,34 +155,36 @@ type:
...
 
     .. code-block:: xml
 
-
         <?xml version="1.0" encoding="UTF-8" ?>
+        <!-- app/config/config.xml -->
```
Comment (wouterj) [inline bundles/routing/configuration.rst]: should be before the doctype

File: bundles/routing/configuration.rst [hunk 4]
```diff
@@ -211,6 +214,7 @@ choose this controller to handle the request.
     .. code-block:: xml
 
         <?xml version="1.0" encoding="UTF-8" ?>
+        <!-- app/config/config.xml -->
```
Comment (wouterj) [inline bundles/routing/configuration.rst]: same here

File: bundles/routing/configuration.rst [hunk 5]
```diff
@@ -264,7 +272,7 @@ setting is set as controller.
 
         <?xml version="1.0" encoding="UTF-8" ?>
         <container xmlns="http://symfony.com/schema/dic/services">
-
+            <!-- app/config/config.xml -->
```
Comment (wouterj) [inline bundles/routing/configuration.rst]: and here

File: bundles/routing/dynamic_customize.rst [hunk 6]
```diff
@@ -61,6 +62,7 @@ is executed.
     .. code-block:: xml
 
         <?xml version="1.0" encoding="UTF-8" ?>
+        <!-- app/config/services.xml -->
```
Comment (wouterj) [inline bundles/routing/dynamic_customize.rst]: ... and here (you get the point, everywhere 😉 )

**Comment by dbu** [comment]
incorporated all this feedback in #804 that is against the correct branch

**Pull request closed without merging**